light/proxy/errors.go

```diff
@@ -0,0 +1,28 @@
+package proxy
+
+import "fmt"
+
+type ErrCreateHTTPClient struct {
+	Addr string
+	Err  error
+}
+
+func (e ErrCreateHTTPClient) Error() string {
+	return fmt.Sprintf("failed to create http client for %s: %v", e.Addr, e.Err)
+}
+
+func (e ErrCreateHTTPClient) Unwrap() error {
+	return e.Err
+}
+
+type ErrStartClient struct {
```

Comment:
```suggestion
type ErrStartHTTPClient struct {
```

to be consistent in naming